postgres-exporter/main.libsonnet

```diff
@@ -3,7 +3,7 @@ local k = import 'ksonnet-util/kausal.libsonnet';
 {
   new(
     name,
-    data_source_name='postgresql://$(USER):$(PASSWORD)@$(HOST):$(PORT)/postgres',
+    data_source_uri='postgresql://@$(HOSTNAME):$(PORT)/postgres',
```

Comment:
Should the `@` still be present in the URI if there are no username and password components?